lib/smart_answer_flows/coronavirus-find-support.rb

```diff
@@ -25,6 +25,10 @@ def define
           calculator.need_help_with = response
         end
 
+        validate do
+          calculator.valid_need_help_with?
+        end
+
```

Comment:
Is it possible to use the `none_option` here?

https://github.com/alphagov/smart-answers/blob/master/doc/smart-answers/question-types.md#checkbox_question

Reply:
I.e.
```
 checkbox_question :need_help_with? do
        option :feeling_unsafe
        option :paying_bills
        option :getting_food
        option :being_unemployed
        option :going_to_work
        option :somewhere_to_live
        option :mental_health
        none_option
```

Reply:
Thanks @theseanything - I think this works well, and is possibly closer to the original intended behaviour.